src/components/cylc/Header.vue

```diff
@@ -37,42 +37,47 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         </g>
       </svg>
     </div>
-    <div id="cylc-select-options" class="c-environment-info w-75 d-flex-col justify-center">
-      <v-combobox
-        class="my-3"
-        id="cylc-owner-combobox"
-        :disabled = "store.state.user.user.mode !=='multi user'"
-        bg-color="white"
-        label="server owner"
-        placeholder="server owner"
-        :default="owner"
-        :items="owners"
-        variant="outlined"
-        v-model="owner"
-        @keyup.enter="addOwner(owner)"
-      ></v-combobox>
-      <v-combobox
-        class="my-3"
-        id="cylc-deployment-combobox"
-        :disabled = "store.state.user.user.mode !=='multi user'"
-        bg-color="white"
-        label="deployment"
-        placeholder="deployment"
-        :default="deployment"
-        :items="deployments"
-        variant="outlined"
-        v-model="deployment"
-        @keyup.enter="addDeployment(deployment)"
-      ></v-combobox>
-      <v-btn
-        @click="window.location.href = url.value"
-        class="mx-12 my-2"
-        width="100"
-        color="green"
-        :hidden="isNewRoute == false || store.state.user.user.mode == 'single user'"
-      >
-        Go
-      </v-btn>
+    <div
+      id="cylc-select-options"
+      class="c-environment-info w-100 d-flex flex-column align-center px-4 row-gap-3"
+    >
+      <v-defaults-provider :defaults="{
+        VCombobox: {
+          bgColor: 'white',
+          rules: [(val) => Boolean(val) || 'Required'],
+        }
+      }">
+        <v-combobox
+          class="w-100"
+          id="cylc-owner-combobox"
+          :disabled="store.state.user.user.mode !== 'multi user'"
+          label="server owner"
+          :default="owner"
+          :items="Array.from(owners)"
+          v-model="owner"
+          @keyup.enter="owners.add(owner)"
+        />
+        <v-combobox
+          class="w-100"
+          id="cylc-deployment-combobox"
+          :disabled="store.state.user.user.mode !== 'multi user'"
+          label="deployment"
+          :default="deployment"
+          :items="Array.from(deployments)"
+          v-model="deployment"
+          @keyup.enter="deployments.add(deployment)"
+        />
+        <v-btn
+          v-if="store.state.user.user.mode !== 'single user' && isNewRoute"
+          data-cy="multiuser-go-btn"
+          :href="url"
```

Comment:
Using `v-if` rather than `hidden` means the browser doesn't have to render the DOM node at all, which is generally better for performance.

Also can just set `href` instead of using a click handler.